api_docgen/Format_tutorial.mld

```diff
@@ -265,6 +283,10 @@ When writing a pretty-printing routine, follow these simple rules:
 + Never hesitate to open a box.
 + Output many break hints, otherwise the pretty-printer is in a bad situation
 where it tries to do its best, which is always "worse than your bad".
++ Break hints with negative indentation should only be used when you are sure
+that you don't exceed the indentation of the opening box (e.g. both part of
+the same function and an exact match). Exceeding the indentation of the opening
+box would not be compositional.
```

Comment:
I would be firmer here since we are in the midst of an imperative sentence: `would not be compositional` ⇒ `is not compositional`.